src/dmd/statement.d

```diff
@@ -1788,6 +1789,7 @@ extern (C++) final class GotoDefaultStatement : Statement
 extern (C++) final class GotoCaseStatement : Statement
 {
     Expression exp;     // null, or which case to goto
+
```

Comment:
?

Reply:
I like to separate the fields set by the constructor and those that come later. It can't be a hard and fast rule because sometimes optimal packing and caching requires a different order.

Reply:
Fair enough as long as there is method to the madness. you might want to make note of that somewhere so people later don't undo it.